src/biogeochem/CNPhenologyMod.F90

```diff
@@ -944,16 +1015,7 @@ subroutine CNSeasonDecidPhenology (num_soilp, filter_soilp       , &
                   if (days_active(p) > 355._r8) pftmayexist(p) = .false.
                end if
 
-               if ( min_crtical_dayl_depends_on_lat )then
```

Comment:
it seems like some of these comments may be helpful either in the code or when the parameters are set

Reply:
I'm adding these comments to the top of the code where these parameter are defined.